lightning/src/ln/channel.rs

```diff
@@ -4419,6 +4419,9 @@ pub(super) struct FundedChannel<SP: Deref> where SP::Target: SignerProvider {
 	pub context: ChannelContext<SP>,
 	pub interactive_tx_signing_session: Option<InteractiveTxSigningSession>,
 	holder_commitment_point: HolderCommitmentPoint,
+	/// Indicates whether this funded channel had been established with V2 channel
+	/// establishment (i.e. is a dual-funded channel).
+	is_v2_established: Option<()>,
```

Comment:
Can this be inferred from the `ChannelId`?

Reply:
Yeah I realised it would probably be the case we could as we could just generate a V2 channel ID again and see if it matches. Unfortunately lost train of though this past week.